cdk/package.json

```diff
@@ -9,8 +9,7 @@
     "lint": "eslint lib/** bin/** --ext .ts --no-error-on-unmatched-pattern",
     "lint:fix": "npm run lint -- --fix",
     "synth": "cdk synth --path-metadata false --version-reporting false",
-    "diff": "cdk diff --path-metadata false --version-reporting false",
-    "difflocal": "cdk diff FaciaPurger-PROD --template ../facia-purger/cloudformation.yaml --path-metadata false --version-reporting false "
+    "diff": "cdk diff --path-metadata false --version-reporting false"
   },
   "devDependencies": {
     "@guardian/cdk": "47.0.1",
```

Comment:
Will update this to the [latest version](https://github.com/guardian/cdk/releases) in a following PR.